topofileformats/asd.py

```diff
@@ -35,7 +40,7 @@ def __init__(self, analogue_digital_range, max_voltage, scaling_factor, resoluti
         self.max_voltage = max_voltage
         self.scaling_factor = scaling_factor
         self.resolution = resolution
-        print(
+        logger.info(
             f"created voltage converter. ad_range: {analogue_digital_range} -> {self.ad_range}, \
```

Comment:
If lines are too long for linting I think you can avoid the need to escape line breaks with...

```suggestion
            f"created voltage converter. ad_range: {analogue_digital_range} -> {self.ad_range}, "
            "max voltage: {max_voltage}, scaling factor: {scaling_factor}, resolution: {resolution}"
```

Not sure one approach is inherently better than another though. :thinking:

Reply:
Done, thank you!